resources/views/applicant/application_post/step_01/questions.html.twig

```diff
@@ -112,7 +112,54 @@
 
             </div>
 
-        {# Language #}
+        {# Language Requirement Declaration:
+            - Only one of these checkboxes appears on any given job application.
+            - This checkbox MUST be checked in order to submit the application.
+        #}
+        <div class="application-post__language-declaration">
+            <span>{{ application_template.language_declaration }}</span>
+            <div class="wrapper">
+            {% if job.language_requirement.id == 1 %}
```

Comment:
@xander-miller This is more of a change in approach to building these views, but in the future I would try and "prepare" as much conditional logic as possible within the controller method, rather than having ~30 lines of duplicated html conditionally selected within the view.

This leads to maintainability issues in the future, where one or more sections are missed during a refactor or frontend markup change.

Reply:
Understood, Tristan had the same feedback. I didn't do that part, I believe Josh did but I didn't check. In the future if I see something like that I will refactor it into the controller.